lib/src/mouse.dart

```diff
@@ -14,37 +14,51 @@
 
 part of webdriver.core;
 
-class Mouse extends _WebDriverBase {
-  static const int left = 0;
-  static const int middle = 1;
-  static const int right = 2;
+enum MouseButton {
+  // the order of elements in this enum is important, as per
+  // https://w3c.github.io/uievents/#widl-MouseEvent-button
+
+  /// The primary button is usually the left button or the only button on
+  /// single-button devices, used to activate a user interface control or select
+  /// text.
+  primary, // = 0
```

Comment:
I don't think it's a good idea to use Dart enums in this case, where it is mandatory that the elements have a specific value. Dart enums aren't built for this purpose.